Enigmatry.Entry.BlobStorage/IPrivateBlobStorage.cs

```diff
@@ -1,10 +1,13 @@
-﻿using JetBrains.Annotations;
+﻿using Enigmatry.Entry.BlobStorage.Models;
+using JetBrains.Annotations;
 
 namespace Enigmatry.Entry.BlobStorage;
 
 [PublicAPI]
 public interface IPrivateBlobStorage : IBlobStorage
 {
-    string BuildSharedResourcePath(string relativePath, string? fileName = null, PrivateBlobPermission permission = PrivateBlobPermission.Read);
+    [Obsolete("Use BuildSharedResourcePath with ContentDispositionSettings instead.")]
+    string BuildSharedResourcePath(string relativePath, string fileName, PrivateBlobPermission permission = PrivateBlobPermission.Read);
```

Comment:
This method is obsolete, so it will be called only if user specified second parameter `fileName`.